src/Operator/Constructors.jl

```diff
@@ -8,7 +8,8 @@
 
 """
 ```julia
-GalleryOperator(name, numbernodes1d, numberquadraturepoints1d, mesh)
+GalleryOperator(name, numbernodes1d, numberquadraturepoints1d, mesh,
+collocatedquadrature, mapping)
```

Comment:
Not sure if the formatting will change in linting, but we should callout the keyword args like we did for the bases
```suggestion
GalleryOperator(
    name,
    numbernodes1d,
    numberquadraturepoints1d,
    mesh;
    collocatedquadrature = false,
    mapping = nothing
)
```